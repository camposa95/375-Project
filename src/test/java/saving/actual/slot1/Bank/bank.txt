ORE: 19
WOOL: 19
LUMBER: 19
BRICK: 19
GRAIN: 19
