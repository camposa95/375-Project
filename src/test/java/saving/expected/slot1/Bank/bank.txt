BRICK: 17
WOOL: 19
LUMBER: 15
ORE: 17
GRAIN: 15
